nexus/src/authz/omicron.polar

```diff
@@ -230,8 +230,29 @@ has_relation(fleet: Fleet, "parent_fleet", collection: ConsoleSessionList)
 # read silo users and modify their sessions.  This is necessary for login to
 # work.
 has_permission(actor: AuthenticatedActor, "read", user: SiloUser)
-	if has_role(actor, "external-authenticator", user.fleet);
+	if has_role(actor, "external-authenticator", user.silo.fleet);
 has_permission(actor: AuthenticatedActor, "read", session: ConsoleSession)
 	if has_role(actor, "external-authenticator", session.fleet);
 has_permission(actor: AuthenticatedActor, "modify", session: ConsoleSession)
 	if has_role(actor, "external-authenticator", session.fleet);
+
+resource SiloUser {
+	permissions = [ "read", "modify", "create_child" ];
```

Comment:
Any reason not to have "list_children" here like other resources, and treating ssh keys like those children?

Reply:
Nope, no reason; fixed in 95114e05.